hat/assets/js/apps/Iaso/libs/Api.ts

```diff
@@ -68,6 +68,9 @@ export const iasoFetch = async (
         return emptyRes;
     }
     if (!response.ok) {
+        if (response.status === 401) {
+            window.location.href = '/login';
+        }
```

Comment:
You should probably add a `return` in the `if` since the case has been handled.

```suggestion
        if (response.status === 401) {
            window.location.href = '/login';
            return;
        }
```

Reply:
Also, do you think it could be possible to pass the current route as a query parameter?
Like, if you are currently on `"/form"` the window location would be:
`/login?redirect_to=/form`